.evergreen/serverless/create-instance.sh

```diff
@@ -22,6 +22,14 @@ set +o xtrace # Disable xtrace to ensure credentials aren't leaked
 #   SERVERLESS_URI            SRV connection string for newly created instance
 #   SERVERLESS_INSTANCE_NAME  Name of newly created instance (required for "get" and "delete" scripts)
 
+SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+. $SCRIPT_DIR/../handle-paths.sh
+
+# Ensure that secrets have already been set up.
+if [ -f "$SCRIPT_DIR/secrets-export.sh" ]; then 
```

Comment:
What if this file doesn't exist?

Reply:
Legacy invocations won't have the file, the individual env var checks below will handle it.